apps/discordbot/bot.py

```diff
@@ -171,18 +182,10 @@ def check_thumb(react, _):
         await self.test_channel.send(f"{member} read rules")
 
         await member.send(
-            "Our mail servers are down right now, so unfortunately we can't verify you yet. Stay "
-            "tuned for updates in the #announcements channel!"
+            "Please verify your berkeley.edu email to gain access to the server! "
```

Comment:
Specify that the user must send a DM to Philbot. There may be confusion with where to send `!verify`

Reply:
Oops, just saw this. Changed in new commit.